crucible-jvm/tool/Main.hs

```diff
@@ -110,40 +111,34 @@ defaultOptions =
   , mainMethod = "main"
   }
 
-cruxJVM :: Crux.Language JVMOptions
-cruxJVM = Crux.Language
-  { Crux.name = "crux-jvm"
-  , Crux.version = "0.1"
-  , Crux.configuration = Crux.Config
-      { Crux.cfgFile = pure defaultOptions
-      , Crux.cfgEnv =
-          [ Crux.EnvVar "JDK_JAR"
-            "Path to .jar file containing the JDK"
-            $ \p opts -> Right $ opts { jarList = p : jarList opts }
-          ]
-      , Crux.cfgCmdLineFlag =
-          [ Crux.Option ['c'] ["classpath"]
-            "TODO"
-            $ Crux.ReqArg "TODO"
-            $ \p opts ->
-                Right $ opts { classPath = classPath opts ++ splitSearchPath p }
-          , Crux.Option ['j'] ["jars"]
-            "TODO"
-            $ Crux.ReqArg "TODO"
-            $ \p opts ->
-                Right $ opts { jarList = jarList opts ++ splitSearchPath p }
-          , Crux.Option ['m'] ["method"]
-            "Method to simulate"
-            $ Crux.ReqArg "method name"
-            $ \p opts -> Right $ opts { mainMethod = p }
-          ]
-      }
-  , Crux.initialize = return
-  , Crux.simulate = simulateJVM
-  , Crux.makeCounterExamples = \_ _ -> return ()
+cruxJVMConfig :: Crux.Config JVMOptions
+cruxJVMConfig = Crux.Config
+  { Crux.cfgFile = pure defaultOptions
+  , Crux.cfgEnv =
+      [ Crux.EnvVar "JDK_JAR"
+        "Path to .jar file containing the JDK"
+        $ \p opts -> Right $ opts { jarList = p : jarList opts }
+      ]
+  , Crux.cfgCmdLineFlag =
+      [ Crux.Option ['c'] ["classpath"]
+        "TODO"
```

Comment:
Heh. We should probably document these properly at some point. I'll create an issue.